README.md

```diff
@@ -68,26 +68,34 @@ cd orbit.<your_extension_name>
 mv orbit/ext_template orbit/<your_extension_name>
 ```
 
-- Set up a symbolic link from Orbit to this directory.
-This makes it convenient to index the python modules and look for extensions shipped with Isaac Sim and Orbit.
+- Define the following environment variable to specify the path to your Orbit installation:
 
 ```bash
-ln -s <your_orbit_path> _orbit
+# Set the ORBIT_PATH environment variable to point to your Orbit installation directory
+export ORBIT_PATH=<your_orbit_path>
+
+# Append the ORBIT_PATH variable definition to your .bashrc file for automatic setup in future sessions
+echo -e "\nexport ORBIT_PATH=${ORBIT_PATH}" >> $HOME/.bashrc
```

Comment:
Is it possible to not have these variables in the bashrc? It really gets messed up if you have multiple orbit versions in your system. For instance, I have three right now (development public, private and personal projects)...

Reply:
Yess, this was only used for `.vscode/settings.json`. I found away around this in the new commit, where users will add the absolute Orbit path as an argument when running `Tasks: Run Task` -> `setup_python_env`